src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -653,11 +1092,11 @@ private void ScanInterpolatedStringLiteralNestedString()
                 _lexer.ScanStringLiteral(ref info, inDirective: false);
             }
 
-            private void ScanInterpolatedStringLiteralHoleBracketed(char start, char end)
+            private void ScanInterpolatedStringLiteralHoleBracketed(InterpolatedStringKind kind, char start, char end)
             {
                 Debug.Assert(start == _lexer.TextWindow.PeekChar());
                 _lexer.TextWindow.AdvanceChar();
-                ScanInterpolatedStringLiteralHoleBalancedText(end, isHole: false, colonRange: out _);
+                ScanInterpolatedStringLiteralHoleBalancedText(kind, end, isHole: false, colonRange: out _);
```

Comment:
Not related to this PR: consider adding an example of interpolation with more than one $ sign in the summary section of the spec: [https://github.com/dotnet/csharplang/blob/main/proposals/raw-string-literal.md](https://github.com/dotnet/csharplang/blob/main/proposals/raw-string-literal.md)

Reply:
will do!

Reply:
Added.